### DISCUSSION TITLE
Airplay 2 Multi-Room Remote Volume Command

### DISCUSSION THREAD

**Discussion by zhjb7** [mikebrady/shairport-sync#1490] [Q&A]
I am playing AP2 in ubuntu, and with one homepod. I send VolumeDown via the Shairport Sync D-Bus interface, this will also down the volume of homepod. Can I use any other protocol command to just down my ubuntu? Thank!

**Comment by mikebrady** [comment]
Thanks. There is nothing built in to Shairport Sync to allow this to happen, I'm afraid.
